apps/api/lib/validations/booking.ts

```diff
@@ -58,6 +58,7 @@ export const schemaBookingReadPublic = Booking.extend({
       })
     )
     .optional(),
+  responses: z.record(z.any()),
```

Comment:
It seemed like this could have been added a level up at the  `Booking` which this is extending but it seemed safer to add it here since this is only used by the `/bookings` API